package.json

```diff
@@ -7,9 +7,9 @@
         "dev": "nodemon ./index.js"
     },
     "dependencies": {
-        "@arkecosystem/crypto": "^2.6.0-next.1",
-        "@arkecosystem/core-utils": "^2.6.0-next.1",
-        "@arkecosystem/core-magistrate-crypto": "^2.6.0-next.1"
+        "@arkecosystem/crypto": "^2.6.0-next.4",
```

Comment:
Those bumps aren't necessary as `^` indicates that it should pick any release that matches `^2.6.0-next.X` where `X` has to be higher than the previous value. You can simply run `yarn install` or `yarn upgrade` to update to the latest version as long as it is in the `^2.6.0-next.X` range.

Reply:
Gotcha. I've reverted the commit :+1: